### PR TITLE
Set use for links (either to kRamp or kTurnChannel)

### PR DESCRIPTION
**Pull request by dnesbitt61**
Turn channels are short, never motorway_link or trunk_link, have no exit information (TODO), and each end node connects to a non-link (e.g. neither end node connects only to links - indicating a split or fork).

### CODE REVIEW

File: src/mjolnir/graphbuilder.cc [hunk 1]
```diff
@@ -927,6 +977,9 @@ void GraphBuilder::BuildLocalTiles(const uint8_t level) const {
     }
   }
 
+  std::cout << "Ramp count = " << rampcount << " Turn Channel count = " <<
```
Comment (kevinkreiser) [inline src/mjolnir/graphbuilder.cc]: use logging

File: src/mjolnir/graphbuilder.cc [hunk 2]
```diff
@@ -737,6 +770,23 @@ void BuildTileSet(std::unordered_map<GraphId, std::vector<uint64_t> >::const_ite
           directededge.set_bridge(w.bridge());
           directededge.set_bikenetwork(w.bike_network());
 
+          // Link - this indicates a ramp or a turn channel. If link is set
+          // test to see if we can infer that the edge is a turn channel
+          directededge.set_link(w.link());
+          if (directededge.link()) {
+            if (directededge.use() != Use::kNone) {
+              std::cout << "Edge is a link but has use = " <<
```
Comment (kevinkreiser) [inline src/mjolnir/graphbuilder.cc]: use logging

**Comment by kevinkreiser** [comment]
does anyone else feel like graphbuilder is getting to big for its britches? i feel like we should pull out attribution/tag processing to another file and there also seem to be a ton of "helper" functions in there. might they be better suited moving into some of the objects they operate on?

**Comment by kevinkreiser** [comment]
after swapping out cout for LOG_, ship

**Comment by gknisely** [comment]
ship after logging

**Pull request merged**